projects/public-search/src/app/patron-profile/patron-profile-personal-editor/patron-profile-personal-editor.component.html

```diff
@@ -0,0 +1,32 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2022 RERO
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<div class="container pl-0">
+  <form [formGroup]="form" (ngSubmit)="submit()">
+    <div class="col">
+      <formly-form class="col mb-0" [model]="model" [fields]="fields" [form]="form"></formly-form>
```

Comment:
two `col` cannot be nested without a `row`: https://getbootstrap.com/docs/3.4/css/#grid-nesting